datalad_debian/new_distribution.py

```diff
@@ -27,6 +27,12 @@
 @build_doc
 class NewDistribution(Interface):
     """Create a new distribution dataset
+
+    A typical distribution dataset contains a 'builder' subdataset with one or
+    more build environments, and a package subdirectory with one subdataset per
+    built Debian package.
```

Comment:
I would remove the word `built`. I can see no purpose in having a `package` subdataset unless we're going to build it. But the subdataset isn't there as a result of building.

Reply:
makes sense, thx